srtcore/epoll.cpp

```diff
@@ -369,6 +313,49 @@ int CEPoll::update_ssock(const int eid, const SYSSOCKET& s, const int* events)
    return 0;
 }
 
+int CEPoll::uwait(const int eid, SRT_EPOLL_EVENT* fdsSet, int fdsSize, int64_t msTimeOut, bool edgeMode)
+{
+    int64_t entertime = CTimer::getTime();
+    if (!fdsSet && fdsSize)
+        throw CUDTException(MJ_NOTSUP, MN_INVAL);
+
+    while (true)
+    {
+        {
+            CGuard pg(m_EPollLock);
+            map<int, CEPollDesc>::iterator p = m_mPolls.find(eid);
+            if (p == m_mPolls.end())
+                throw CUDTException(MJ_NOTSUP, MN_EIDINVAL);
+            if (p->second.m_sUDTSocksWait.empty() && p->second.m_sLocals.empty() && (msTimeOut < 0))
```

Comment:
Provided that the `uwait` version doesn't exit on checking readiness on system sockets, shouldn't the `m_sLocals` be even **required** to be empty?